File: C++/B-ch1-4-開國皇帝的平均壽命-取小數兩位.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
    int life[9] = {50, 62, 55, 64, 70, 50, 80, 71, 52};
    int i, sum = 0;
    float average;

    for (i=0; i<9; i++) {
        sum = sum + life[i];
    }
    
    average = sum / 9.0;
    average = round(average * 100) /100;
    cout <<  average;
    
    return 0;
}

File: Python/b_ch1_4_開國皇帝的平均壽命-取小數兩位.py
life = [50, 62, 55, 64, 70, 50, 80, 71, 52]
sum = 0
num = len(life)
for i in range(num):
  sum = sum + life[i]

average = sum / num
average = round(average * 100) / 100
print(average)